src/content/docs/en/guides/styling.mdx

```diff
@@ -148,6 +148,11 @@ This pattern lets you style child components directly. Astro will pass the paren
 Because the `class` prop includes the child in its parent’s scope, it is possible for styles to cascade from parent to child. To avoid this having unintended side effects, ensure you use unique class names in the child component.
 :::
 
+### Style objects can be used inline instead of strings
+
+When styling components in Astro, you may use inline style objects. For example: `<div  style={{ '--cssVariable': 'brown' }} >`
+
+When you run Astro , this will be transformed into the following: `<div style="--css-variable:brown" class="astro-J7PV25F6">`
```

Comment:
You can style HTML elements inline using the `style` attribute. This can be a CSS string or an object of CSS properties:

```astro
//These are equivalent:
<p style={{ color: "brown", textDecoration: "underline" }}>My text</p>
<p style="color: brown; text-decoration: underline">My text</p>
```